spec/controllers/imprint_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+describe ImprintsController, type: :controller do
```

Comment:
Controller tests are quite useless.

What we want to test is a feature for printing an order:
1. User logging in.
2. Navigates to specific order.
3. Clicks "print" button.
4. Gets file!

Improve this test and move it to features (spec/features/orders/). And name it print_order_spec.rb.